targets/cloudscript-typings/make.js

```diff
@@ -26,3 +28,122 @@ exports.makeServerAPI = function (apis, sourceDir, apiOutputDir) {
     writeFile(path.resolve(outputDir, "CloudScript.d.ts"), generatedApi);
 
 }
+
+/**
+  Handles selecting the correct datatype template to use for a datatype.
+  enum, interface
+*/
+function MakeDatatype(datatype, api, sourceDir) {
+  var stringLiteralTemplate = compileTemplate(sourceDir,"StringLiteral");
+  var interfaceTemplate = compileTemplate(sourceDir,"Interface");
+
+  var locals = {
+    name: datatype.name,
+  };
+
+  if(datatype.isenum) {
```

Comment:
(Alternately, you could fetch the templates once, and store them as a global since you're going to call them so many times)